pkg/convert/json_ngxtranslate.go

```diff
@@ -0,0 +1,72 @@
+package convert
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/expect-digital/translate/pkg/model"
+)
+
+var ErrWrongKey = errors.New("Wrong key type")
+
+func fromNgxTranslate(b []byte) (messages model.Messages, err error) {
+	var dst map[string]interface{}
+
+	if err = json.Unmarshal(b, &dst); err != nil {
+		return messages, fmt.Errorf("unmarshal ngx translate: %w", err)
+	}
+
+	traverseMap := func(key string, value any) (err error) {
+		return err
+	}
+
+	traverseMap = func(key string, v any) (err error) {
+		switch value := v.(type) {
+		default:
+			return ErrWrongKey
+		case string:
+			messages.Messages = append(messages.Messages, model.Message{ID: key, Message: value})
+		case map[string]interface{}:
+			for key2, val2 := range value {
+				var newKey string
+				if key != "" {
+					newKey += key + "."
+				}
+
+				newKey += key2
```

Comment:
Keep it simple, you can still keep it clear and nice without declaring a new variable.

Alternative names `subKey`, `subValue` to `key2`, `value2`.

```suggestion
				if key != "" {
					subKey = key + "." + subKey
				}
```